__tests__/tests.test.js

```diff
@@ -157,4 +157,83 @@ describe.only("/api/articles/:article_id/comments", () => {
         expect(body.msg).toBe("Bad request");
       });
   });
+  test("GET 404: responds with a 404 status code when requesting a comment from an author that exists but has no comments", () => {
+    return request(app)
+      .get("/api/articles/4/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article does not exist");
+      });
+  });
+});
+describe("/api/articles/:article_id/comments", () => {
+  test("POST 201: responds with a 201 status code and adds the new comment to the existing database, then returns to user what was added", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "sam approves this message",
+    };
+    return request(app)
+      .post("/api/articles/6/comments")
+      .send(newComment)
+      .expect(200)
+      .then(({ body }) => {
+        body = body.comment;
+        expect(body.article_id).toBe(6);
+        expect(typeof body.author).toBe("string");
+        expect(typeof body.body).toBe("string");
+        expect(typeof body.created_at).toBe("string");
+        expect(typeof body.votes).toBe("number");
+      });
+  });
+  test("POST 404: responds with a 404 status code and returns custom error", () => {
```

Comment:
in your describe - you should explain the behaviour that you do as well (i.e. 'responds with a 404 status when the article_id is valid but non-existent')